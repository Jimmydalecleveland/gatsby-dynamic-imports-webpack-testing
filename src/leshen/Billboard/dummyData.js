import lifestyle from '../../images/split-content-lifestyle.jpg'
import lifestyleFull from '../../images/billboard-lifestyle-full.jpg'

export const FLUID_IMAGE_DATA = {
  context: {
    custom: {
      alt: 'Alt Text',
    },
  },
  fluid: {
    base64:
      'data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAPABQDASIAAhEBAxEB/8QAGAAAAwEBAAAAAAAAAAAAAAAAAAQFAQL/xAAVAQEBAAAAAAAAAAAAAAAAAAABAv/aAAwDAQACEAMQAAABY4VxKBIGf//EABoQAAIDAQEAAAAAAAAAAAAAAAABAgMREjL/2gAIAQEAAQUCrjpP1qE3lksOz//EABYRAQEBAAAAAAAAAAAAAAAAAAAREv/aAAgBAwEBPwHSv//EABURAQEAAAAAAAAAAAAAAAAAAAAR/9oACAECAQE/AYj/xAAcEAACAQUBAAAAAAAAAAAAAAAAATECEBJBUWH/2gAIAQEABj8C6JUySZ6F7b//xAAcEAACAgIDAAAAAAAAAAAAAAABEQAxIVEQQZH/2gAIAQEAAT8hIHQaigsQF4gXT2NWQVuEdhW3x//aAAwDAQACAAMAAAAQ4+//xAAXEQADAQAAAAAAAAAAAAAAAAAAAREh/9oACAEDAQE/EMVUo//EABYRAQEBAAAAAAAAAAAAAAAAAAARUf/aAAgBAgEBPxDaH//EAB4QAQACAgEFAAAAAAAAAAAAAAERIQAxcUFRYZGx/9oACAEBAAE/EEKI60I86xyRZUEjqPvNJHwy9miG2LXgZRab+9sUtAHGf//Z',
    src: lifestyle,
    srcSet: lifestyle,
    aspectRatio: 1.3524904214559388,
    sizes: '(max-width: 706px) 100vw, 706px',
    presentationWidth: 2560,
  },
}

export const FLUID_IMAGE_DATA_FULL = {
  context: {
    custom: {
      alt: 'Alt Text',
    },
  },
  fluid: {
    base64:
      'data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAGABQDASIAAhEBAxEB/8QAFgABAQEAAAAAAAAAAAAAAAAAAAMC/8QAFQEBAQAAAAAAAAAAAAAAAAAAAAH/2gAMAwEAAhADEAAAAdULJA//xAAZEAACAwEAAAAAAAAAAAAAAAABAgADBBL/2gAIAQEAAQUCTYTBa1kfXZ3/AP/EABQRAQAAAAAAAAAAAAAAAAAAABD/2gAIAQMBAT8BP//EABURAQEAAAAAAAAAAAAAAAAAABAR/9oACAECAQE/AYf/xAAcEAACAQUBAAAAAAAAAAAAAAAAEQECEiExQXH/2gAIAQEABj8C0i6ZxTwlJeH/xAAZEAADAQEBAAAAAAAAAAAAAAABESEAQVH/2gAIAQEAAT8hN23nNcLWQCawQJjHn//aAAwDAQACAAMAAAAQBB//xAAWEQEBAQAAAAAAAAAAAAAAAAABEBH/2gAIAQMBAT8QXJ//xAAWEQEBAQAAAAAAAAAAAAAAAAABABH/2gAIAQIBAT8QA2X/xAAaEAEAAgMBAAAAAAAAAAAAAAABACERMWGR/9oACAEBAAE/EGQY0bPrcwQjrko72algtuO3P//Z',
    src: lifestyleFull,
    srcSet: lifestyleFull,
    aspectRatio: 3.116883116883117,
    sizes: '(max-width: 800px) 100vw, 800px',
    presentationWidth: 2560,
  },
}
