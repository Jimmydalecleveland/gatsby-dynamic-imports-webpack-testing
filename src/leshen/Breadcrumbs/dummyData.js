export const CRUMB_DATA = [
  {
    shortName: 'Home',
    path: '/',
  },
  {
    shortName: 'Internet',
    path: '/internet',
  },
  {
    shortName: 'Gigabit',
    path: '/gigabit',
  },
  {
    shortName: 'Gigabit Plans',
    path: '/gigabit-plans',
  },
]
