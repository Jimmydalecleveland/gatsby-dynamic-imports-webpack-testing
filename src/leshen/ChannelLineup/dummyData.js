const CHANNEL_LINEUP_DATA = {
  channels: {
    edges: [
      {
        node: {
          name: 'History Channel',
          packageAvailability: [
            'dn-americastop120',
            'dn-americastop120+',
            'dn-americastop200',
            'dn-americastop250',
          ],
          brandy_id: 'history-channel',
        },
      },
      {
        node: {
          name: 'Nickelodeon ',
          packageAvailability: [
            'dn-americastop120',
            'dn-americastop120+',
            'dn-americastop200',
            'dn-americastop250',
          ],
          brandy_id: 'nickelodeon',
        },
      },
      {
        node: {
          name: 'FX',
          packageAvailability: [
            'dn-americastop120',
            'dn-americastop120+',
            'dn-americastop200',
            'dn-americastop250',
          ],
          brandy_id: 'fx',
        },
      },
      {
        node: {
          name: 'ESPN',
          packageAvailability: [
            'dn-americastop120',
            'dn-americastop120+',
            'dn-americastop200',
            'dn-americastop250',
          ],
          brandy_id: 'espn',
        },
      },
      {
        node: {
          name: 'FOX',
          packageAvailability: [
            'dn-americastop120',
            'dn-americastop120+',
            'dn-americastop200',
            'dn-americastop250',
          ],
          brandy_id: 'fox',
        },
      },
      {
        node: {
          name: 'CNN',
          packageAvailability: [
            'dn-americastop120',
            'dn-americastop120+',
            'dn-americastop200',
            'dn-americastop250',
          ],
          brandy_id: 'cnn',
        },
      },
      {
        node: {
          name: 'Cartoon Network',
          packageAvailability: [
            'dn-americastop120',
            'dn-americastop120+',
            'dn-americastop200',
            'dn-americastop250',
          ],
          brandy_id: 'cartoon-network',
        },
      },
      {
        node: {
          name: 'Comedy Central',
          packageAvailability: [
            'dn-americastop120',
            'dn-americastop120+',
            'dn-americastop200',
            'dn-americastop250',
          ],
          brandy_id: 'comedy-central',
          url: '/tv/channels/comedy-central',
        },
      },
      {
        node: {
          name: 'Nickelodeon / Nick At Nite - West (NICKW)',
          packageAvailability: [
            'dn-americastop120',
            'dn-americastop120+',
            'dn-americastop200',
            'dn-americastop250',
          ],
          brandy_id: 'nickelodeon',
        },
      },
      {
        node: {
          name: 'Turner Network Television (TNT)',
          packageAvailability: [
            'dn-americastop120',
            'dn-americastop120+',
            'dn-americastop200',
          ],
          brandy_id: 'tnt',
          url: '/tv/channels/tnt',
        },
      },
      {
        node: {
          name: 'TV Land (TVLND)',
          packageAvailability: ['dn-americastop120+', 'dn-americastop250'],
          brandy_id: 'tvland',
        },
      },
      {
        node: {
          name: 'E! Entertainment Television (E!)',
          packageAvailability: [
            'dn-americastop120',
            'dn-americastop120+',
            'dn-americastop200',
          ],
          brandy_id: 'e',
        },
      },
      {
        node: {
          name: 'Syfy Channel (SYFY)',
          packageAvailability: [
            'dn-americastop120',
            'dn-americastop120+',
            'dn-americastop200',
            'dn-americastop250',
          ],
          brandy_id: 'syfy',
        },
      },
      {
        node: {
          name: 'Eternal Word Television Network (EWTN)',
          packageAvailability: [
            'dn-americastop120',
            'dn-americastop120+',
            'dn-americastop200',
            'dn-americastop250',
          ],
          brandy_id: 'ewtn',
        },
      },
    ],
  },
  packages: {
    edges: [
      {
        node: {
          name: "America's Top 120",
          brandy_id: 'dn-americastop120',
        },
      },
      {
        node: {
          name: "America's Top 120+",
          brandy_id: 'dn-americastop120+',
        },
      },
      {
        node: {
          name: "America's Top 200",
          brandy_id: 'dn-americastop200',
        },
      },
      {
        node: {
          name: "America's Top 250",
          brandy_id: 'dn-americastop250',
        },
      },
    ],
  },
}

export default CHANNEL_LINEUP_DATA
