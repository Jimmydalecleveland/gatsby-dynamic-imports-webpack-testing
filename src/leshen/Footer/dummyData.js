export const FOOTER_NAV_ITEMS = [
  {
    text: 'Services',
    href: false,
    subnav: [
      {
        text: 'Clearlink Internet',
        href: '#',
      },
      {
        text: 'FiOS Internet',
        href: '#',
      },
      {
        text: 'Bundles',
        href: '#',
      },
      {
        text: 'Frontier TV',
        href: '#',
      },
      {
        text: 'Dish TV',
        href: '#',
      },
      {
        text: 'Clearlink Phone',
        href: '#',
      },
      {
        text: 'Business',
        href: '#',
      },
    ],
  },
  {
    text: 'About',
    href: false,
    subnav: [
      {
        text: 'Store Locations',
        href: '#',
      },
      {
        text: 'Media Center',
        href: '#',
      },
      {
        text: 'Scholarship',
        href: '#',
      },
      {
        text: 'Contact Us',
        href: '#',
      },
    ],
  },
  {
    text: 'Español',
    href: false,
    subnav: [
      {
        text: 'Internet',
        href: '#',
      },
      {
        text: 'FiOS Internet',
        href: '#',
      },
      {
        text: 'DishLATINO',
        href: '#',
      },
      {
        text: 'Teléfono',
        href: '#',
      },
      {
        text: 'Paquetes',
        href: '#',
      },
    ],
  },
  {
    text: 'Terms',
    href: false,
    subnav: [
      {
        text: 'Legal',
        href: '#',
      },
      {
        text: 'Privacy Policy',
        href: '#',
      },
      {
        text: 'Terms of Service',
        href: '#',
      },
    ],
  },
]

export const DISCLAIMERS = [
  '©2019 Internet Service Partners. All Rights Reserved. The name CenturyLink and the pathways logo are trademarks of CenturyLink.',
  '*Geographic and service restrictions apply to AT&T services. Call to see if you qualify. Fiber optics may apply to all or part of the network, depending on your location.',
  '© 2019 AT&T Intellectual Property. AT&T, the AT&T logo and all other AT&T marks contained herein are trademarks of AT&T Intellectual Property and/or AT&T affiliated companies. All rights reserved. Subsidiaries and affiliates of AT&T Inc. provide products and services under the AT&T brand.',
]
