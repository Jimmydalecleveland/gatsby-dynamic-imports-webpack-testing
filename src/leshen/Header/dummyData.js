export const HEADER_DATA = {
  mainNavItems: [
    {
      text: 'FiOS',
      subnav: [
        { text: 'Clearlink FiOs', href: '/' },
        { text: 'What is FiOs?', href: '/' },
        { text: 'FiOs vs Cable', href: '/' },
      ],
    },
    {
      text: 'Internet',
      href: '/BillboardPage',
      megaMenu: false,
      subnav: [
        { text: 'FiOs Internet', href: '/' },
        { text: 'High Speed Internet', href: '/' },
        { text: 'Vantage Internet?', href: '/' },
      ],
    },
    { text: 'TV', href: '/TabsPage' },
    { text: 'Phone', href: '/' },
    { text: 'Plans & Pricing', href: '/' },
    { text: 'More', href: 'https://github.com' },
  ],

  topBarNavItems: [{ text: 'Clearlink Business', href: '/' }],
}

export const HEADER_MEGAMENU_DATA = {
  mainNavItems: [
    {
      text: 'FiOS',
      href: '/',
      subnav: [
        { text: 'Clearlink FiOs', href: '/' },
        { text: 'What is FiOs?', href: '/' },
        { text: 'FiOs vs Cable', href: '/' },
      ],
    },
    {
      text: 'Internet',
      href: '/BillboardPage',
      megaMenu: true,
      subnav: [
        {
          text: 'FiOS Internet',
          href: '/',
          subnav: [
            { text: 'Clearlink FiOS', href: '/' },
            { text: 'What is FiOS?', href: '/' },
            { text: 'FiOS vs Cable?', href: '/' },
            { text: 'Who am I?', href: '/' },
          ],
        },
        {
          text: 'High Speed Internet',
          href: '/',
          subnav: [
            {
              text: 'What is Internet?',
              href: '/',
              subnav: [
                {
                  text: 'What even is it?',
                  href: '/',
                },
                {
                  text: 'Series of Tubes?',
                  href: '/',
                },
                {
                  text: 'Best Speeds',
                  href: '/',
                  subnav: [
                    { text: 'Fiber Internet', href: '/' },
                    { text: 'Recommended Daily Fiber', href: '/' },
                  ],
                },
              ],
            },
            { text: 'How Do I Internet?', href: '/' },
            { text: 'Cuantos Internets?', href: '/' },
          ],
        },
        {
          text: 'Vantage Internet?',
          href: '/',
          subnav: [
            { text: 'Im Stuck In The Internet', href: '/' },
            { text: 'Can you help me?', href: '/' },
            { text: 'Vantage Advantage?', href: '/' },
          ],
        },
      ],
    },
    { text: 'TV', href: '/TabsPage' },
    { text: 'Phone', href: '/' },
    { text: 'Plans & Pricing', href: '/' },
    { text: 'More', href: 'https://github.com' },
  ],

  topBarNavItems: [{ text: 'Clearlink Business', href: '/' }],
}
