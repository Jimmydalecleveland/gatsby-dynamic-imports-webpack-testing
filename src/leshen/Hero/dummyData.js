import hero from "../images/hero.jpg"
import lifestyle from "../images/split-content-lifestyle.jpg"

export const FLUID_IMAGE_DATA = {
  context: {
    custom: {
      alt: "Alt Text",
    },
  },
  fluid: {
    base64:
      "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAJABQDASIAAhEBAxEB/8QAFwAAAwEAAAAAAAAAAAAAAAAAAAQFA//EABUBAQEAAAAAAAAAAAAAAAAAAAAB/9oADAMBAAIQAxAAAAG9IagJsJB//8QAHBAAAQMFAAAAAAAAAAAAAAAAAQACAwQRMTIz/9oACAEBAAEFAlVSNIMgu/o7Mu//xAAUEQEAAAAAAAAAAAAAAAAAAAAQ/9oACAEDAQE/AT//xAAUEQEAAAAAAAAAAAAAAAAAAAAQ/9oACAECAQE/AT//xAAZEAEAAgMAAAAAAAAAAAAAAAABABASIYH/2gAIAQEABj8CgE3nxor/xAAaEAACAwEBAAAAAAAAAAAAAAAAAREhMXGh/9oACAEBAAE/IVg/i1cjso2x04eU0P/aAAwDAQACAAMAAAAQC8//xAAWEQEBAQAAAAAAAAAAAAAAAAAAARH/2gAIAQMBAT8QrH//xAAUEQEAAAAAAAAAAAAAAAAAAAAQ/9oACAECAQE/ED//xAAcEAEAAQUBAQAAAAAAAAAAAAABABEhMWGhsfD/2gAIAQEAAT8QsS4pGC3QMLp1LO2KgBic7yfJtnEeT//Z",
    src: hero,
    srcSet: hero,
    aspectRatio: 2.3529411764705883,
    sizes: "(max-width: 800px) 100vw, 800px",
    presentationWidth: 800,
  },
}

export const FLUID_IMAGE_DATA_SPLIT = {
  context: {
    custom: {
      alt: "Alt Text",
    },
  },
  fluid: {
    base64:
      "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAPABQDASIAAhEBAxEB/8QAGAAAAwEBAAAAAAAAAAAAAAAAAAQFAQL/xAAVAQEBAAAAAAAAAAAAAAAAAAABAv/aAAwDAQACEAMQAAABY4VxKBIGf//EABoQAAIDAQEAAAAAAAAAAAAAAAABAgMREjL/2gAIAQEAAQUCrjpP1qE3lksOz//EABYRAQEBAAAAAAAAAAAAAAAAAAAREv/aAAgBAwEBPwHSv//EABURAQEAAAAAAAAAAAAAAAAAAAAR/9oACAECAQE/AYj/xAAcEAACAQUBAAAAAAAAAAAAAAAAATECEBJBUWH/2gAIAQEABj8C6JUySZ6F7b//xAAcEAACAgIDAAAAAAAAAAAAAAABEQAxIVEQQZH/2gAIAQEAAT8hIHQaigsQF4gXT2NWQVuEdhW3x//aAAwDAQACAAMAAAAQ4+//xAAXEQADAQAAAAAAAAAAAAAAAAAAAREh/9oACAEDAQE/EMVUo//EABYRAQEBAAAAAAAAAAAAAAAAAAARUf/aAAgBAgEBPxDaH//EAB4QAQACAgEFAAAAAAAAAAAAAAERIQAxcUFRYZGx/9oACAEBAAE/EEKI60I86xyRZUEjqPvNJHwy9miG2LXgZRab+9sUtAHGf//Z",
    src: lifestyle,
    srcSet: lifestyle,
    aspectRatio: 1.3524904214559388,
    sizes: "(max-width: 706px) 100vw, 706px",
    presentationWidth: 706,
  },
}
