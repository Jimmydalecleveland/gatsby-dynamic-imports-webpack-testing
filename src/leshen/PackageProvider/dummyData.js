// eslint-disable-next-line import/prefer-default-export
export const PACKAGE_DATA = {
  name: 'DIRECTVÂ® Ultimate All-Included + INTERNET + HOME PHONE',
  details: ['300 Channels', '30Mbps Internet', 'Home Phone'],
  bullets: [
    { text: 'Speeds up to 20 Mbps' },
    { text: 'No contract' },
    {
      text: 'No rate hike',
      disclaimer: {
        text:
          'For 12 months with 24-mo. DIRECTV agreement and combined billing. Price includes DIRECTV ULTIMATE All- Included Package ($60.00/mo.)',
      },
    },
  ],
  price: {
    amount: '79.99',
    disclaimer: {
      text:
        'For 12 months with 24-mo. DIRECTV agreement and combined billing. Price includes DIRECTV ULTIMATE All- Included Package ($60.00/mo.)',
    },
  },
  legal:
    'For 12 months with 24-mo. DIRECTV agreement and combined billing. Price includes DIRECTV ULTIMATE All- Included Package ($60.00/mo.)',
}
