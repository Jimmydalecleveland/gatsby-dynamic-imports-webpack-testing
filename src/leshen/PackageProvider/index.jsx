import React from 'react'
import PropTypes from 'prop-types'
import { Done } from 'emotion-icons/material'

import Brandy from '../Brandy'
import Typography from '../Typography'
import Price from '../Price'
import List from '../List'
import ListItem from '../List/ListItem'

const PackageProvider = ({ name, price, termsLink, details, bullets }) => (
  <>
    {name && (
      <Typography variant="h5" className="title">
        {name}
      </Typography>
    )}
    {details && (
      <div className="details">
        {details.map((detail) => (
          <span key={detail}>{detail}</span>
        ))}
      </div>
    )}
    {price && price.amount && (
      <Price className="price" {...price}>
        {price.disclaimer && (
          <Brandy
            symbol={price.disclaimer.symbol}
            text={price.disclaimer.text}
            variant="legal"
          />
        )}
      </Price>
    )}
    {bullets.length > 0 && (
      <List>
        {bullets.map((bullet) => (
          <ListItem key={bullet.text} icon={<Done />}>
            <Typography>{bullet.text}</Typography>
            {bullet.disclaimer && (
              <Brandy variant="legal" text={bullet.disclaimer.text} />
            )}
          </ListItem>
        ))}
      </List>
    )}
    {termsLink && (
      <a href="/terms" className="terms">
        {termsLink}
      </a>
    )}
  </>
)

PackageProvider.defaultProps = {
  name: '',
  termsLink: '',
  legal: null,
  details: [],
  bullets: [],
}

PackageProvider.propTypes = {
  name: PropTypes.string,
  price: PropTypes.shape({
    label: PropTypes.string,
    amount: PropTypes.string.isRequired,
    disclaimer: PropTypes.shape({
      symbol: PropTypes.string,
      text: PropTypes.string.isRequired,
    }),
  }).isRequired,
  legal: PropTypes.node,
  termsLink: PropTypes.string,
  // TODO: use callback proptype the check the length > 3
  details: PropTypes.arrayOf(PropTypes.string),
  bullets: PropTypes.arrayOf(
    PropTypes.shape({
      text: PropTypes.string.isRequired,
      disclaimer: PropTypes.shape({
        symbol: PropTypes.string,
        text: PropTypes.string.isRequired,
      }),
    })
  ),
}

export default PackageProvider

export const ticketID = 'CMS-149'
