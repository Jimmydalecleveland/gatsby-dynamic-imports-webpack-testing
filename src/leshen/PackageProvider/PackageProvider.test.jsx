import React from 'react'
import { render } from 'setup-tests'
import PackageProvider from '.'
import { PACKAGE_DATA } from './dummyData'

describe('PackageProvider', () => {
  test('PackageProvider should render', () => {
    const { container } = render(<PackageProvider {...PACKAGE_DATA} />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
