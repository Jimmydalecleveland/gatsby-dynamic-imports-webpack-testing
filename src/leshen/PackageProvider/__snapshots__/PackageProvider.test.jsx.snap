// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PackageProvider PackageProvider should render 1`] = `
.emotion-0 {
  font-weight: 500;
  font-size: calc(0.5555555555555556vw + 17.333333333333332px);
  line-height: calc(1.1111111111111112vw + 18.666666666666668px);
}

@media (max-width:480px) {
  .emotion-0 {
    font-size: 20px;
  }
}

@media (min-width:1200px) {
  .emotion-0 {
    font-size: 24px;
  }
}

@media (max-width:480px) {
  .emotion-0 {
    line-height: 24px;
  }
}

@media (min-width:1200px) {
  .emotion-0 {
    line-height: 32px;
  }
}

@media all and (-ms-high-contrast:none),(-ms-high-contrast:active) {
  .emotion-0 {
    width: 100%;
  }
}

<h5
  class="title emotion-0 emotion-1"
>
  DIRECTVÂ® Ultimate All-Included + INTERNET + HOME PHONE
</h5>
`;
